angel-morales/tarea-2/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="css/index.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portada de Revista</title>
+  </head>
+  <body>
+    <main class="container">
+      <header class="content-title" title="titular de noticia">
+        <h3 class="subtitle">Futbol de Alto Nivel</h3>
+        <h1 class="title">
+          La joya <span class="color-title">Colombiana</span>
+        </h1>
+      </header>
+
+      <article class="content-page">
+        <aside class="aside-left">
+          <div class="text-content-left">
+            <p class="text-content">
+              Golazo de Luis Díaz enloqueció a los hinchas del Liverpool
+            </p>
+          </div>
+          <div class="text-content-left">
```

Comment:
@AngelMrles7 this div should be a section

Reply:
Good morning.
I have already made the changes